sympy/solvers/pde.py

```diff
@@ -567,7 +572,17 @@ def pde_1st_linear_constant_coeff(eq, func, order, match, solvefun):
     where `a`, `b` and `c` are constants and `G(x, y)` can be an arbitrary
     function in `x` and `y`.
 
-    The general solution of the PDE is::
+    The general solution of the PDE is:
+
+    .. math::
+        f(x, y) = \left. \left[F(\eta) + \frac{1}{a^2 + b^2}
```

Comment:
What is F here?

Reply:
According to the reference is an arbitrary single-valued function.

Reply:
Can you explain that in the docstring?

Reply:
I added a comment about this in the docstrings.